<?xml version="1.0"?>
<launch>

    <!-- if running on real robot or in simulation_only -->
    <arg name="simulation_only" default="true" />
    <param name="simulation_only" value="$(arg simulation_only)" />

    <!-- positional error that is added to model position to test robustness of algorithm -->
    <arg name="pos_error_x" default="0.0" />
    <arg name="pos_error_y" default="0.0" />
    <arg name="pos_error_z" default="0.0" />
    <param name="pos_error_x" value="$(arg pos_error_x)" />
    <param name="pos_error_y" value="$(arg pos_error_y)" />
    <param name="pos_error_z" value="$(arg pos_error_z)" />

    <!-- intial tool pose in spherical coordinates. initial wrist pose is calculated from these. -->
    <arg name="polar" default="0.0" />
    <arg name="azimuthal" default="0.0" />
    <arg name="offset" default="0.3" />
    <param name="polar" value="$(arg polar)" />
    <param name="azimuthal" value="$(arg azimuthal)" />
    <param name="offset" value="$(arg offset)" />

    <!-- name of log file where experimental results get added -->
    <arg name="log_name" default="experiments.csv" />
    <param name="log_name" value="$(arg log_name)" />

    <!-- if the hand doesn't make a grasp attempt until 'time_out' seconds, then stop experiment -->
    <arg name="time_out" default="15" />
    <param name="time_out" value="$(arg time_out)" />

    <!-- run baseline_commander node -->
    <node pkg="brain" type="baseline_controller_node" name="baseline_controller_node" output="screen"/>

</launch>
